buddyboss/buddyboss-platform
# Enhancement Update the Forum reply count, post count, scrubber count, and pagination to be the same --- For Theme 2.0

### All Submissions:






### Changes proposed in this Pull Request:



Fixes https://github.com/buddyboss/buddyboss-theme/issues/1851

### How to test the changes in this Pull Request:
https://www.loom.com/share/103aa16a96014786a9b285dea12bc6fe

1.    In forum listing, the number of replies should be exactly the same number as in the admin: https://www.loom.com/share/69e3326e576a4d35a34a4bd6e85676bb

2.    In single discussion, the Posts count should be renamed to Replies and it should be exactly the same number as in the admin: https://www.loom.com/share/0701884075644ea0b6a3cb4aac2cabd8

3.    In single discussion, the pagination should be actually paginating by reply count, and should be the correct number of replies: https://www.loom.com/share/03c8d2a5a45a4b6ab4ddbfd345815ecb

4.    In single discussion, change the scrubber text from "1 of 22 posts" to "1 of 22 replies". And then update the numbers to be inline with pagination, so like "1 of 31 replies" on top of page 1, "15 of 31 replies" at bottom of page 1, then "16 of 31 replies" on top of page 2, and then "31 of 31 replies at bottom of page 2, as an example. Ignore the first post which is not a reply. https://www.loom.com/share/bf8e77861e724d24989370eb5fbd49eb

5.    We also need to look at the forums widgets and shortcodes and make sure they all are corrected along with this, to have correct reply counts and pagination etc.


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
